backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/SubscriptionsController.java

```diff
@@ -98,6 +102,44 @@ public SubscriptionModel getSubscriptionById(
 		return subscriptionModelMapper.toModel(userId, subscription);
 	}
 
+	@PutMapping({ "/{subscriptionId}" })
+	@Operation(summary = "Update a subscriptions by ID")
+	@ApiResponse(responseCode = "204", description = "The request has been successfully processed.")
+	public void updateSubscriptionById(
+			@NotBlank(message = "userId must not be null or blank")
+			@Parameter(description = "The id of the user.", example = "00000000-0000-0000-0000-000000000000")
+			@PathVariable String userId,
+
+			@NotBlank(message = "subscriptionId must not be null or blank")
+			@Parameter(description = "The id of the subscription.", example = "00000000-0000-0000-0000-000000000000")
+			@PathVariable String subscriptionId,
+			
+			@Validated @RequestBody PreferredLanguageModel preferredLanguageModel) {
+		
+		userService.updateSubscriptionForUser(userId, subscriptionId, preferredLanguageModel);
+	}
+
+	@DeleteMapping({ "/{subscriptionId}" })
+	@Operation(summary = "delete a subscriptions by ID")
+	@ApiResponse(responseCode = "204", description = "The request has been successfully processed.")
+	public void deleteSubscriptionById(
+			@NotBlank(message = "userId must not be null or blank")
+			@Parameter(description = "The id of the user.", example = "00000000-0000-0000-0000-000000000000")
+			@PathVariable String userId,
+
+			@NotBlank(message = "subscriptionId must not be null or blank")
+			@Parameter(description = "The id of the subscription.", example = "00000000-0000-0000-0000-000000000000")
+			@PathVariable String subscriptionId) {
+		final var user = userService.getUserById(userId)
+			.orElseThrow(() -> new ResourceNotFoundException("No user with id=[%s] was found".formatted(userId)));
+
+		final var subscription = user.getSubscriptions().stream()
+			.filter(byId(subscriptionId)).findFirst()
+			.orElseThrow(() -> new ResourceNotFoundException("No subscription with id=[%s] was found".formatted(subscriptionId)));
+
+		userService.deleteSubscriptionForUser(userId, subscription);
```

Comment:
```suggestion
		userService.deleteSubscriptionForUser(userId, subscriptionId);
```

should be sufficient here.. allow the caller to send the minimal amount of data in the simplest form that is required to complete the operation.

Reply:
fixed